OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/Optimove.java

```diff
@@ -71,6 +72,7 @@ final public class Optimove {
 
     private Optimove(@NonNull Context context) {
         this.context = context;
+
         this.coreSharedPreferences = context.getSharedPreferences(TenantConfigsKeys.CORE_SP_FILE,
```

Comment:
can initialize all of these only if optimoveConfig given. Then it would fail earlier if try to use optimove methods. but then these cannot be final